### PR TITLE
Add ability to customize slack report frequency

### PR DESCRIPTION
**Pull request by msabramo**
via `SLACK_DAILY_REPORT_FREQUENCY` env var

Addresses one thing that @taralika mentioned in https://github.com/BerriAI/litellm/issues/3597

### CODE REVIEW

**Comment by ghost** [comment]
can you add this to docs - https://docs.litellm.ai/docs/proxy/alerting#step-1-add-a-slack-webhook-url-to-env

**Comment by msabramo** [comment]
> can you add this to docs - https://docs.litellm.ai/docs/proxy/alerting#step-1-add-a-slack-webhook-url-to-env

Yep. Done in 94f735359.
Preview at https://github.com/msabramo/litellm/blob/abramowi/customizable-slack-report-frequency/docs/my-website/docs/proxy/alerting.md#step-2-update-configyaml

File: docs/my-website/docs/proxy/alerting.md [hunk 1]
```diff
@@ -33,13 +34,11 @@ general_settings:
     alerting: ["slack"]
     alerting_threshold: 300 # sends alerts if requests hang for 5min+ and responses take 5min+ 
 
+environment_variables:
+    SLACK_WEBHOOK_URL: "https://hooks.slack.com/services/<>/<>/<>"
+    SLACK_DAILY_REPORT_FREQUENCY: "86400"  # 24 hours; Optional: defaults to 12 hours
```
Comment (ishaan-jaff) [inline docs/my-website/docs/proxy/alerting.md:47]: please can you add more detail on what the daily reports are?  A user might be unsure about what this param means / controls

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]

File: docs/my-website/docs/proxy/alerting.md (hunk 1 above)
Comment (msabramo) [reply docs/my-website/docs/proxy/alerting.md:47]: Sure! How about 464545114?

Comment (ishaan-jaff) [reply docs/my-website/docs/proxy/alerting.md:47]: awesome ! thanks

**Review by ishaan-jaff — APPROVED** [review]

File: docs/my-website/docs/proxy/alerting.md (hunk 1 above)
Comment (msabramo) [reply docs/my-website/docs/proxy/alerting.md:47]: Screenshot 2024-05-13 at 9 06 56 PM

**Comment by ghost** [comment]
LGTM!

**Pull request merged**